packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/constants.js

```diff
@@ -32,3 +32,7 @@ export const STAGE_COLOR_DEFAULT = lightTheme.colors.primary600;
 export const DRAG_DROP_TYPES = {
   STAGE: 'stage',
 };
+
+export const CHARGEBEE_WORKFLOW_ENTITLEMENT_NAME = 'review-workflows-number-of-workflows';
+export const CHARGEBEE_STAGES_PER_WORKFLOW_ENTITLEMENT_NAME =
+  'review-workflows--stages-per-workflow';
```

Comment:
should this have two dashes or one? if two, why different from the above?

Reply:
weird

Reply:
🤷🏼